### DISCUSSION TITLE
How do you deploy the github-pages-environment?

### DISCUSSION THREAD

**Discussion by cstenkamp** [frankier/skelshop#52] [Q&A]
Hey Frankie,

For another project I'd like to do the documentation similar to how you did it here, but I don't understand how you're building it. Actually, I've never even seen the "Environments" section in the right side of the project in any other project. How do you deploy the docs automatically? I don't see any github-workflows or the like, the only CD I see at all is the `.gitlab.ci.yml`, however that one seems to only run tests, and not deploy any documentation.
I've looked around and found github-workflows like this https://github.com/marketplace/actions/github-pages, however none of them make the `Environments` section appear, and apparently you don't use any github Actions at all?

**Comment by frankier** [comment]
Hi! Sorry for the delay with answering this. It slipped my mind. The way mkdocs works is that you can just deploy straight from your machine. So I just run `poetry run mkdocs gh-deploy`. I didn't like it first time I saw it because it didn't seem very reproducible, but it turns out to be fine for simple cases.

**Reply by frankier** [reply]
I don't know if that answers it. I just followed https://www.mkdocs.org/user-guide/deploying-your-docs/ . I think it shows up when you enable GitHub pages for a repository.
